wormchain/contracts/tools/deploy_wormchain.ts

```diff
@@ -42,6 +42,9 @@ const readFileAsync = util.promisify(fs.readFile);
 type ContractName = string;
 const artifacts: ContractName[] = [
   "global_accountant.wasm",
+  "cw_wormhole.wasm",
+  "cw20_wrapped_2.wasm",
+  "cw_token_bridge.wasm",
```

Comment:
Changing the order here may change the addresses (which might cause some inconsistencies). We should probably double check the IBC receiver didn't change.